srcpkgs/siril/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'siril'
+pkgname=siril
+version=0.9.12
+revision=1
+archs="x86_64"
+build_style=gnu-configure
+configure_script=./autogen.sh
```

Comment:
The usual way to do this (if the script supports it) is to run autogen.sh with NOCONFIGURE=1 in a `pre_configure` function and then let xbps-src use the created configure script with the default configuration.

Reply:
Fixed